python_modules/dagster/dagster/_core/execution/asset_backfill.py

```diff
@@ -713,6 +733,15 @@ def _submit_runs_and_update_backfill_in_chunks(
                 submitted_partitions
             )
         )
+        if code_unreachable_error_raised:
+            # Code server became unavailable mid-backfill. Rewind the cursor back to the cursor
+            # from the previous iteration, to allow next iteration to reevaluate the same
+            # events.
```

Comment:
Every 30s, the daemon searches for all in-progress backfills and loops through each one, executing an asset backfill iteration for each.

So if a backfill had a code location error, it would be re-evaluated for on every daemon "loop". But on each loop, all the other backfills could successfully execute.